config/environments/development.rb

```diff
@@ -8,7 +8,27 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = true
+  config.cache_classes = false
+
+  # Note: This is a hack to prevent warnings like:
+  #
+  #   warning: already initialized constant LogMessages::Conjur::PrimarySchema
+  #   warning: previous definition of PrimarySchema was here
+  #
+  # These warnings only appear when "config.cache_classes" is set to "false",
+  # as it is above.  Suppressing them shouldn't be a problem in development.
+  #
+  module Warning
+    CONJUR_IGNORE_RE = Regexp.union(
+      /previous definition of [^ ]* was here/,
+      /already initialized constant/
+    )
+
+    def warn(msg)
+      return if msg =~ CONJUR_IGNORE_RE
+      super(msg)
+    end
+  end
```

Comment:
@jonahx what is the benefit of this change? don't we want to see these errors in DEV so we can verify they are not showing in PROD? 

I am worried that some warnings will slip to PROD without us noticing.